queenbee/schema/arguments.py

```diff
@@ -206,6 +209,12 @@ class Arguments(BaseModel):
         'task or workflow.'
     )
 
+    user_data: Dict = Field(
```

Comment:
Can we call this `labels` or `tags` rather than user data? If the intent is for them to be used as reference material I think those works ,make more sense.

Reply:
Also, I understand what we're trying to achieve with this in the sense that an individual simulation should be able to label itself with additional metadata. However it really rubs me the wrong way that we would put this in the `arguments` schema. This is because arguments are used everywhere in both input, outputs, individual task template etc... Could we put this at the workflow level and make it another variable that a `simulation trigger` payload would impact?

Reply:
The reason I put it here is that one can update the values in an argument using an input file. Maybe I should create a separate type only for workflow inputs which can have `user_data` field.

I thought we should keep the tag as a list for workflow. For instance it will be daylighting, energy, etc. `user_data` has a different use case than tags. I'm down with using labels which will match k8 terminology I guess.

Reply:
I added a new object type for WorkflowArgument which will only be used as a workflow input. It has an extra field for user_data which is a dictionary.

I will add tags and remove the default value for id in a different PR.